sdk/resourcemanager/Azure.ResourceManager/docs/MigrationGuide.md

```diff
@@ -599,6 +599,61 @@ VirtualMachineResource virtualMachine = virtualMachineOperation.Value;
 
 Finally, as it can be seen here, from the resource group you can get the Virtual Machine collection and create a new one using the `VirtualMachineData` for the parameters.
 
+### Update a Virtual Machine
+
+**Create a new object of VirtualMachineData**
+```C#
```

Comment:
and we should use the `snippets` to verify the code here could properly compile.
Please refer to how it is done for other code snippets in this file (for instance search `Create_Fluent_VirtualMachine` in this file), and the snippets could be found here: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/samples/DocSamples/MigrationGuide.cs#L253

You need to:
1. Create a new method in the file above, wrap the part that you would like to show up here in the md file with a `#region Snippet:SnippetName` and `#endregion`
2. Add `Snippet:SnippetName` on this line next to the `C#` (please taking other snippets as references)
3. Remove the code here
4. Run the script: `<Root>/eng/scripts/Update-Snippets.ps1 resourcemanager` to update the code to this md file.

Reply:
resolved